### PR TITLE
Setup base styles

### PR DESCRIPTION
**Pull request by betoobernal**
## Summary
- Added styles base to the project setup

## Trello Card
https://trello.com/c/0E3H0OSP/11-setup-del-proyecto

### CODE REVIEW

File: scss/styles.scss [hunk 1]
```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: You should create a `colors.scss` file and import it in `application.scss`

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+
+body {
```
Comment (mmamaniwolox) [inline scss/styles.scss]: use `*` instead, you dont want this style in body, you want this to reach every kind of html tag

File: scss/styles.scss [hunk 3]
```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: texts ones should go to `fonts.scss`

File: scss/styles.scss [hunk 4]
```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$nav-height: 70px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: You can create a `titles.scss` also

File: scss/styles.scss [hunk 5]
```diff
@@ -1,3 +1,20 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
+
+$nav-height: 70px;
```
Comment (mmamaniwolox) [inline scss/styles.scss]: you will use this ones when you create the `navbar.scss` file

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: scss/styles.scss (hunk 4 above)
Comment (betoobernal) [reply scss/styles.scss]: I will remove it. And I will add it in the page-title PR

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**